src/main/java/net/syscon/elite/repository/impl/OffenderBookingIdSeq.java

```diff
@@ -0,0 +1,20 @@
+package net.syscon.elite.repository.impl;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class OffenderBookingIdSeq {
+    private String offenderNo;
+    private Long bookingId;
```

Comment:
Could add an Optional getter here for bookingId forcing the isPresent syntax instead of != null...meh.

Reply:
@Mjwillis It might be worth thinking about this comment. It is sort of thing that will catch people out, and because we rarely have offenders without bookings it will never get tested, so making it Optional  and forcing the developer to handle nulls is a very good idea.